yoga/model/assimp.h

```diff
@@ -29,6 +29,7 @@ typedef enum OutputFormat {
 typedef enum OptimizationFlag {
     OPTIMIZATION_FLAG_GRAPH = 1,
     OPTIMIZATION_FLAG_MESHES = 2,
+    FIX_FLAG_NORMALS = 4,
```

Comment:
Does not look very conventional.
In C, we tend to prefix the enum values with the name of the enum itself.

I suggest renaming the enum `Flag` and have:
```C
typedef enum Flag {
    FLAG_OPTIMIZE_GRAPH = 1,
    FLAG_OPTIMIZE_MESHES = 2,
    FLAG_FIX_INFACING_NORMALS = 4,
} OptimizationFlag;
```